### PR TITLE
fix #382 add imports to the Readme and update usage of HeadOption

### LINKED ISSUE #382: Please add imports to README
The lack of imports in the README means that users have to go hunting to figure out where to get the functionality. Please add them!

### PR DESCRIPTION
**Pull request by julien-truffaut**
@nuttycom Could you double check?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,14 +107,13 @@ object `A` and in our case, the first character of a `String` is optional as a `
 we need a sort of partial `Lens`, in Monocle it is called `Optional`.
 
 ```scala
-import monocle.function.headOption._ // to use headOption (a generic optic)
-import monocle.std.string._          // to get String instance for HeadOption
+import monocle.function.Cons.headOption // to use headOption (a generic optic)
+  import monocle.std.string._             // to get String instance for HeadOption
```
Comment (aoiroaoino) [inline README.md]: indent?

Comment (julien-truffaut) [inline README.md]: thanks

**Comment by aoiroaoino** [comment]
LGTM 👍

**Comment by aoiroaoino** [comment]
@nuttycom ping ✨ 
if you have any good ideas, pull request are welcome. 😄

**Pull request merged**